package.json

```diff
@@ -14,45 +14,45 @@
     "generate-api-schema": "node utils/generateAPISchema.js"
   },
   "dependencies": {
-    "@google-cloud/firestore": "^4.9.1",
+    "@google-cloud/firestore": "^4.9.4",
     "axios": "^0.21.1",
-    "config": "^3.3.2",
-    "cookie-parser": "~1.4.4",
+    "config": "^3.3.3",
+    "cookie-parser": "~1.4.5",
     "cors": "^2.8.5",
-    "debug": "~2.6.9",
-    "express": "~4.16.1",
+    "debug": "~4.3.1",
+    "express": "~4.17.1",
     "express-boom": "^3.0.0",
-    "helmet": "^4.0.0",
-    "http-errors": "~1.6.3",
-    "joi": "^17.2.1",
+    "helmet": "^4.4.1",
+    "http-errors": "~1.8.0",
+    "joi": "^17.4.0",
     "jsdoc": "^3.6.6",
     "jsonwebtoken": "^8.5.1",
-    "morgan": "~1.9.1",
+    "morgan": "~1.10.0",
     "passport": "^0.4.1",
     "passport-github2": "^0.1.12",
-    "swagger-jsdoc": "^4.0.0",
-    "swagger-ui-express": "^4.1.4",
+    "swagger-jsdoc": "^6.0.6",
```

Comment:
@swarajpure , have you verified that the upgrading to the packages like swagger-jsdoc and eslint-config-standard, there are no breaking changes, as its a major version upgrade?

Post that, we can merge the PR.

Reply:
We're having a new warning:
![image](https://user-images.githubusercontent.com/56217868/109023637-f92b8f00-76e2-11eb-9feb-211622dcb42e.png)

Other than that, I tried to generate api schema, checked swagger docs, they look fine to me. Not sure how to check for jsdoc
Is there any proper way to check for breaking changes?

Reply:
@swarajpure , can you fix the new warnings occurring due to the upgraded version, if those can be fixed?

You can go to the changelog for that package and view it there. Check if any of the change affects us. I suppose there might nor be any change in the schema generation.

Changelog: https://github.com/Surnet/swagger-jsdoc/releases

Semantic versioning: https://github.com/Surnet/swagger-jsdoc/releases. This might help in understanding the versioning...

Reply:
Will fix the warning 👍🏻 

Yes I had seen that link already and comparing between our current package and the latest stable package just shows me some commits 🤔

Reply:
Fixed the warning.